### ISSUE TITLE
Current issues

### ISSUE THREAD

**Issue by arminask** [arminask/windows-arm-apps#125]
- ~~Website load time.~~
(Unrelated, issue was on my end)

- [Resources page ](https://armrepo.ver.lt/#resources)layout/design needs to be improved
I don't particularly like the way it looks right now, what do others think?
This page could also include other docs to ARM related things.

- Inconsistencies with the application subtitles
I currently find the software labeling a bit messy. Some apps have architecture labels, some do not, 
some have the software description, some don't. I think agreeing on a standardized subtitle formatting would
be more consistent across all the  apps?

These are just my thoughts written, so that I don't forget. You're welcome to agree or disagree on them.

**Comment by talynone** [comment]
For me the website loads instantly, but I have a pretty good Internet connection. The png files are already pretty small, you could try re-write the website so it only loads images on demand when scrolled into. 

https://css-tricks.com/lazy-loading-images-with-vue-js-directives-and-intersection-observer/

**Comment by lexcyn** [comment]
@arminask I agree with the subtitles. This would be very beneficial if they had a tag saying native, ARM64EC or even web app (neutral arch).

The page loads pretty instant for me but I am on 1.5Gbps fiber :)

**Comment by arminask** [comment]
My internet is 1Gbps up/down fiber. Strangely enough I cannot replicate this issue anymore. It probably was my browser/cache or an Ad Blocking extension.

**Comment by iKineticate** [comment]
I've noticed that within some categories, there are software entries with similar functionalities that are not grouped together. For example:

- Software 1 and Software 2 both offer [specific functionality], yet they are listed separately.

- Grouping these similar software together could enhance user experience and make it easier to compare options.

I suggest considering a reorganization of these entries to enhance clarity and user experience.
